config/manifests/bases/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -434,19 +433,16 @@ spec:
     adoption of cloud native workloads, improved productivity and scalable operations.
```

Comment:
Hi Rensy, the first paragraph in the description refers to a specific CSI driver and only mentioned 1/2 modules. 
We need to refresh that portion as well and keep it generic. I think we should only reference as CSM modules or drivers and not reference any one particular component. Can you revise that, reach out to me if you have any questions.

Reply:
Done